packages/react-liveness/src/components/FaceLivenessDetector/shared/Hint.tsx

```diff
@@ -83,7 +83,6 @@ export const Hint: React.FC<HintProps> = ({ hintDisplayText }) => {
     [FaceMatchState.CANT_IDENTIFY]: hintDisplayText.hintCanNotIdentifyText,
     [FaceMatchState.FACE_IDENTIFIED]: hintDisplayText.hintTooFarText,
     [FaceMatchState.TOO_MANY]: hintDisplayText.hintTooManyFacesText,
-    [FaceMatchState.TOO_CLOSE]: hintDisplayText.hintTooCloseText,
```

Comment:
Are we still using `hintTooCloseText` elsewhere?

Reply:
Yes but directly (not from the map) https://github.com/aws-amplify/amplify-ui/blob/4366291b170089fbc44191636c2c40b60a77ff69/packages/react-liveness/src/components/FaceLivenessDetector/shared/Hint.tsx#L127